bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx126-na.html

```diff
@@ -8,7 +8,7 @@
 
 {% extends "firefox/whatsnew/base.html" %}
 
-{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_title %}{{ ftl('whatsnew-page-title-v2') }}{% endblock %}
 
 {#- This will appear as <meta property="og:description"> which can be used for social share -#}
 {% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
```

Comment:
Would it be easier to manage if all these titles + descriptions from all the wnp templates were [moved to whatsnew/base](https://github.com/mozilla/bedrock/pull/14720#discussion_r1657022464)?

Reply:
Just saw Craig's suggestion -- 
@craigcook is this still something you're hoping to work on?

Reply:
Honestly I forgot about that almost as soon as I posted it... but we've been talking about redoing the base WNP templates for some time and it just keeps getting put off for other work. I do think at the very least we can add the title string to the base file, that doesn't have to wait. If you want to do it as part of this PR that seems sensible.

Reply:
I will be doing some light housekeeping in wnps, welcomes and subnavs soon~ish, so you can leave it like this and i'll pick it up later, possibly together with the v1 nav obsoletes…

(Just wanted to confirm it'd be a welcome change…)

Reply:
That would be a welcome change, thanks Jan!